src/mds/Server.cc

```diff
@@ -349,10 +349,17 @@ void Server::handle_client_session(MClientSession *m)
 	m->put();
 	return;
       }
+      // We are getting a seq that is higher than expected.
+      // Handle the same as any other seqn error.
+      //
       if (m->get_seq() != session->get_push_seq()) {
-	dout(0) << "old push seq " << m->get_seq() << " != " << session->get_push_seq() 
+	dout(0) << "old push seq " << m->get_seq() << " != " << session->get_push_seq()
 		<< ", BUGGY!" << dendl;
-	assert(0);
+	mds->clog->warn() << "incorrect push seq " << m->get_seq() << " != "
+			  << session->get_push_seq() << ", dropping0";
+	mds->clog->warn() << "client unique ID : " << session->info.inst.name << " , client name : " << session->get_human_name() ;
```

Comment:
duh, thanks for seeing the typo.  I'll combine the logs.

On Wed, May 25, 2016 at 8:33 AM, John Spray <notifications@github.com>
wrote:

> In src/mds/Server.cc
> <https://github.com/ceph/ceph/pull/9214#discussion_r64563682>:
>
> >  		<< ", BUGGY!" << dendl;
> > -	assert(0);
> > +	mds->clog->warn() << "incorrect push seq " << m->get_seq() << " != "
> > +			  << session->get_push_seq() << ", dropping0";
> > +	mds->clog->warn() << "client unique ID : " << session->info.inst.name << " , client name : " << session->get_human_name() ;
>
> This should all be in one clog->warn(), otherwise some other message could
> pop up between and make it confusing. You can probably make it a bit less
> verbose, like just add "from client " << human name << " (" << inst.name
> << ")" to the first log line.
>
> Also the "dropping0" typo is still in
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ceph/ceph/pull/9214/files/40740b239640cef860cb28450d894bb06b255f6a#r64563682>
>